### PR TITLE
Add Slack info to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mahalrs**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,6 +73,8 @@ For instructions on reporting security issues and bugs, please see [security][se
 
 For questions about building, running, or troubleshooting, start with the [Developer's Guide][dev-guide], and work your way through the process that we've outlined. If that doesn't answer your question(s), try to post on [Discussion][discussion-link] tab or if you think you found a bug, please file an [issue][issue-link].
```
Comment (nitishm) [inline CONTRIBUTING.md:74]: Fix paragraph indent (seems to be right shifted at least in the review)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -72,6 +73,8 @@ For instructions on reporting security issues and bugs, please see [security][se
...
 
+That said, if you still have question(s), join the Azure Orkestra [Slack](https://join.slack.com/t/azureorkestra/shared_invite/zt-rowzrite-Hm_eaih4GyjjZXWftuoqPQ) and someone will help you get answer(s) to your question(s).
```
Comment (nitishm) [inline CONTRIBUTING.md:76]: ~That said~

**Review by nitishm — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (mahalrs) [reply CONTRIBUTING.md]: Fixed.

**Review by helayoty — APPROVED** [review]
lgtm

File: CONTRIBUTING.md (hunk 1 above)
Comment (mahalrs) [reply CONTRIBUTING.md:74]: It's just in the review.

Comment (mahalrs) [reply CONTRIBUTING.md:74]: Looks good here https://github.com/mahalrs/orkestra/blob/contributing-add-slack/CONTRIBUTING.md#support

**Review by nitishm — COMMENTED** [review]
🚀 🚀

**Review by nitishm — APPROVED** [review]
🚀

**Pull request merged**